tests/test_verify_release_checks.py

```
@@ -0,0 +1,316 @@
+    monkeypatch: pytest.MonkeyPatch, run_id: object
+) -> None:
+    """Fail closed when the dispatch response is not an authoritative run ID.
+
+    Args:
+        monkeypatch (pytest.MonkeyPatch): Fixture for replacing the API helper.
+        run_id (object): Invalid response value.
+    """
+    monkeypatch.setattr(
+        verify,
+        "github_api",
+        lambda _arguments, expected_status=None: {"workflow_run_id": run_id},
```

Ruff (0.16.3)

[warning] Unused lambda argument: `expected_status`

(ARG005)